packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.js

```diff
@@ -1,24 +1,30 @@
 import SfButton from "../SfButton/SfButton.vue";
+import SfIcon from "../SfIcon/SfIcon.vue";
 
 export default {
   name: "SfCircleIcon",
   props: {
     /**
-     * Icon URL
      */
-    icon: {
+    pathIcon: {
+      type: [String, Array],
+      default: "home"
+    },
+    /**
+     */
+    colorIcon: {
       type: String,
-      default: ""
+      default: "white"
     },
     /**
-     * Alt text for icon
      */
```

Comment:
Why you removed alt?

Reply:
Now we use SfIcon ;) and _alt_ is work just img